docs/03.gen.daisy.components.md

```diff
@@ -123,3 +123,29 @@ So, a very simple but viable board description might look like:
 ### Built-in descriptions
 
 If some details are still unclear, the [JSON descriptions for the built-in boards](https://github.com/Wasted-Audio/hvcc/tree/develop/hvcc/generators/c2daisy/json2daisy/resources) might help.
+
+### OLED Display
+
+Some boards like the [Patch](https://github.com/Wasted-Audio/hvcc/blob/develop/hvcc/generators/c2daisy/json2daisy/resources/patch.json) and [Field](https://github.com/Wasted-Audio/hvcc/blob/develop/hvcc/generators/c2daisy/json2daisy/resources/field.json) have an onboard OLED display.
+
+An empty `"display": {}` section triggers inclusion via the default configuration. However by itself it doesn't do anything. You can add several optional settings to use the oled.
+
+- `driver` - Which oled driver to use.
+- `process` - Inline C++ code for the `Display()` function.
+- `process_cpp` - External C++ file with `Display()` function.
```

Comment:
Still not 100% on this name, since both of these are `cpp`.
Maybe `process_file` ?